test/Microsoft.SqlTools.ServiceLayer.UnitTests/Utility/TestObjects.cs

```diff
@@ -282,5 +283,16 @@ public DbConnection CreateSqlConnection(string connectionString, string azureAcc
                 ConnectionString = connectionString
             };
         }
+
+        public SqlConnection ConvertDBConnectionToSqlConnection(DbConnection connection)
+        {
+            return new SqlConnection(connection.ConnectionString);
+        }
+
+        public bool CheckIfContainedAuthorization(DbConnection connection)
+        {
+            //Returning false to mathc the previous behaviour
```

Comment:
This comment doesn't explain anything. What previous behavior? Why does it matter for a test object what the "previous" behavior was?